dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -35,4 +35,13 @@ def post_using_text_mode
 
     render json: {using_text_mode: !!@user.using_text_mode}
   end
+
+  # POST /api/v1/users/<user_id>/post_ui_tip_dismissed
+  def post_ui_tip_dismissed
```

Comment:
There are already tests for `get/post_using_text_mode`, so you should add some additional tests for `post_ui_tip_dismissed`